src/store/item-store.js

```diff
@@ -8,17 +8,24 @@ let items = [
     {
         address: 'iGwEZUvfA',
         modelAddress: 'm8y7nEtAe',
-        status: 'AVAILABLE'
+        status: 'AVAILABLE',
```

Comment:
Why did you add these properties to these items? It's not even all the applicable properties.